notifiers/elasticsearch/elasticsearch.go

```diff
@@ -82,18 +127,22 @@ func Notify(log utils.LogLine) error {
 	return nil
 }
 
-func checkSettings(settings *Settings) error {
-	if settings.URL == "" {
+func checkParameters(parameters *Parameters) error {
```

Comment:
yes we could, this parts have been developed before the use of the validator, because it worked, I prefer to keep it as it now. for sure we'll improve that after the release, for now, a freeze is better to avoid to lose more time.